lib/active_merchant/billing/gateways/digital_river.rb

```diff
@@ -69,6 +69,27 @@ def purchase(options)
         end
       end
 
+      def refund(money, _transaction_id, options)
```

Comment:
I assume the unused var comes from how the refund method is constructed in Conduit?
Could we maybe call the refund method in Conduit without the transaction_id and remove it here?

Reply:
Yeah, this signature is common for all calls to `refund` in AM and Conduit expects it as well.
In Conduit logic for refunds this this transaction_id is passed down a couple of functions which would all require overwritting in the DigitalRiver class. Given the current [WIP PR for Conduit](url) is so small I thought it'd be better not to change the shape of these signatures and leave the variable be.

Reply:
I think I didn't actually link that WIP PR, sorry, updated! 😅